docs/targets/googlecloudpubsub.md

```diff
@@ -0,0 +1,33 @@
+# Event Target for Google Cloud Pub Sub
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to a pre-defined Google Cloud Pub Sub Topic.
+
+
+## Prerequisite(s)
+
+- Google Cloud Console account.
+- A service account and it's associated JSON credentials.
+- a pre-existing Google Cloud Pub Sub Topic.
+
+
+## Deploying an Instance of the Target
+
+1. [Clone][clone] the Triggermesh mono repo and navigate to the example manifest located in the [`triggermesh/config/samples/targets/googlecloudpubsub/`][sample-manifest] section of the Triggermesh mono repo.
```

Comment:
Here and below - camel casing:
```suggestion
1. [Clone][clone] the TriggerMesh mono repo and navigate to the example manifest located in the [`triggermesh/config/samples/targets/googlecloudpubsub/`][sample-manifest] section of the TriggerMesh mono repo.
```

Reply:
Can we also replace `mono repo` with something that does not expose unnecessary technical details of the project? We never used the term `mono repo` for user-facing docs.